packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -34,7 +34,7 @@ class CheckboxGroup extends React.Component {
             name={option.name}
             value={option.value}
             checked={includes(this.props.values, option.value)}
-            onChange={val => this.handleChange(option.value, val)}/>;
+            onChange={e => this.handleChange(option.value, e.target.checked)}/>;
```

Comment:
>  is there some reason why we don't use functional components?

We'd like to preserve the ability to use refs if we need to.

I agree this onChange handler in CheckboxGroup is weird, but that's a separate PR. cf-ui has a number of these  arrow functions inlined into JSX that delegate to some class method handlers with weird signatures.